contracts/contracts/TACoApplication.sol

```diff
@@ -194,7 +196,8 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
         uint256 _minOperatorSeconds,
         uint256 _rewardDuration,
         uint256 _deauthorizationDuration,
-        uint64[] memory _commitmentDurationOptions
+        uint64[] memory _commitmentDurationOptions,
+        uint64 _commitmentDeadline
```

Comment:
Does the deployment script need to be updated?

Reply:
yep, will do
@arjunhassard @cygnusv What date should be for the deadline? like fixed date or should I change it to duration from deployment?
@derekpierre can we get current timestamp in yml?

Reply:
✔️